samples/openapi3/client/petstore/ruby/lib/petstore/api_client.rb

```diff
@@ -379,7 +379,7 @@ def build_collection_param(param, collection_format)
         # return the array directly as typhoeus will handle it as expected
         param
       else
-        fail "unknown collection format: #{collection_format.inspect}"
+        fail "unknown collection format: #{collection_format}"
```

Comment:
I think it's better not to fix it here.
For example, nothing is output when `collection_format` is nil.

Reply:
Sure. Will revert the change as there should be a better fix